sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/DataLakePathCreateOptions.java

```diff
@@ -20,20 +16,15 @@
 @Fluent
 public class DataLakePathCreateOptions {
 
-    private String permissions;
-    private String umask;
+    private DataLakeAccessOptions accessOptions;
+    private DataLakePathScheduleDeletionOptions deletionOptions;
     private PathHttpHeaders headers;
     private Map<String, String> metadata;
     private DataLakeRequestConditions requestConditions;
-    private List<PathAccessControlEntry> accessControlEntryList;
-    private String owner;
-    private String group;
     private String sourceLeaseId;
     private String proposedLeaseId;
     private Long leaseDuration;
-    private PathExpiryOptions expiryOptions;
-    private OffsetDateTime expiresOn;
-    private Duration timeToExpire;
+    private PathExpiryMode expiryOptions;
```

Comment:
I don't think we need this property, we don't have this .NET, and the user can specify the mode by setting either `expiresOn` or `timeToExpire` in `DataLakePathScheduleDeletionOptions`

Reply:
This is a required parameter though for the http headers. Is there some default value we should be passing in then?

Reply:
Make PathExpiryOptions internal so it is not visible to customers, and follow the logic of the .NET SDK https://github.com/Azure/azure-sdk-for-net/blob/03134b8ca8ff040872996daea7abe96add7c5cb9/sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs#L1130